modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManagerTest.java

```diff
@@ -580,24 +617,20 @@ public void testUpdateGroupWithPUTProvidedUserManagerHandlerIsNull(String id, St
 
     @Test(dataProvider = "dataForTestUpdateGroupWithPUTUpdatedGroupResourceIsNull")
     public void testUpdateGroupWithPUTUpdatedGroupResourceIsNull(String id, String scimObjectString, String
-            attributes, String excludeAttributes, Object scimOldGroupObject) {
+            attributes, String excludeAttributes, Object scimOldGroupObject)
+            throws NotImplementedException, BadRequestException, NotFoundException, CharonException {
 
         Group groupOld = (Group) scimOldGroupObject;
         SCIMResourceTypeSchema schema = SCIMResourceSchemaManager.getInstance().getGroupResourceSchema();
-
         abstractResourceManager.when(()
                 -> AbstractResourceManager.getResourceEndpointURL(SCIMConstants.USER_ENDPOINT))
                 .thenReturn(SCIM2_GROUP_ENDPOINT + "/" + GROUP_ID);
         abstractResourceManager.when(()
                 -> AbstractResourceManager.encodeSCIMException(any(InternalErrorException.class)))
                 .thenReturn(getEncodeSCIMExceptionObject(new InternalErrorException()));
-
-        abstractResourceManager.when(() -> userManager.getGroup(id,
+        Mockito.when(userManager.getGroup(id,
                 ResourceManagerUtil.getAllAttributeURIs(schema))).thenReturn(groupOld);
```

Comment:
merge this into 1 line

Reply:
Fixed in commit 34423db0111992d6450b72b5cb1f2b50ab84aec0